process/common_test.go

```diff
@@ -39,16 +40,16 @@ func TestEmptyChannelShouldWorkOnNotBufferedChannel(t *testing.T) {
 	ch := make(chan bool)
 
 	assert.Equal(t, 0, len(ch))
-	readsCnt := process.EmptyChannel(ch)
+	readsCnt := int32(process.EmptyChannel(ch))
 	assert.Equal(t, 0, len(ch))
-	assert.Equal(t, 0, readsCnt)
+	assert.Equal(t, int32(0), readsCnt)
 
 	wg := sync.WaitGroup{}
 	wgChanWasWritten := sync.WaitGroup{}
-	numConcurrentWrites := 100
-	wg.Add(numConcurrentWrites)
-	wgChanWasWritten.Add(numConcurrentWrites)
-	for i := 0; i < numConcurrentWrites; i++ {
+	numConcurrentWrites := int32(100)
```

Comment:
unnecessary cast to int32, should have had the cast only in assert func (less lines modified)

Reply:
Done.